.changeset/hungry-llamas-tie.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-auth-backend': patch
```

Comment:
Can we revert the `auth-backend` changes in this PR? Doesn't look like they are needed at all?

Reply:
Haha yeah sure.

Intellij Idea was showing a warning, i couldn't resist to fix it to get rid of yellow highlight. 
But you are right, it's not a relevant change